.github/workflows/main.yml

```diff
@@ -1,21 +1,22 @@
 # This is a basic workflow to help you get started with Actions
 
 name: CI
-
+ 
 # Controls when the action will run.
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
     branches:
       - master
+      - patch-1
   pull_request:
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
   schedule:
     - cron:  '0 0 * * *'
 env:
-  BASENAME: bitnami/minideb
+  BASENAME: redstonewizard08/minideb
```

Comment:
Is this change expected?

Reply:
It is until production, @javsalgar. This is just until it's merged - I did that for testing.